katas/content/teleportation/reconstruct_message_psi_plus/solution.md

```diff
@@ -0,0 +1,10 @@
+To apply standard teleportation, entanglement pair should be brought back to $\ket{\Phi ^+}$. Correction with `X` gate can help achieve the transformation from $\ket{\Psi ^+}$ to $\ket{\Phi ^+}$.
```

Comment:
```suggestion
Bob's side of this protocol can be represented as a sequence of two steps:
 1. Transform the Bell pair he shares with Alice back to $\ket{\Phi ^+}$. This can be done by applying the $X$ gate to Bob's qubit.
 2. Apply the standard teleportation protocol.
 
The final set of corrections we need is the $X$ gate, followed by the standard teleportation corrections:
```